### DISCUSSION TITLE
[Feature] Grouping faces in folders (family, friends, work, memes, ...) and/or tag people

### DISCUSSION THREAD

**Discussion by Chuckame** [immich-app/immich#6258] [Feature Request]
### The feature

Currently, all the faces are in the same ui. But it would be great to separate them, like into folders.

Why I had this idea ? Because I have too much faces recognized because of whatsapp memes, so I would like to quickly access to family when searching for family people per example.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by schuhbacca** [comment]
As an FYI you can also set the min recognized faces parameter to be higher to remove one off faces recognized.

**Reply by Chuckame** [reply]
Thanks for the info, I'll maybe increase it a little bit.

But still, my family is really big, I meet a lot of people, it would be awesome to group people! 😁

**Comment by bo0tzz** [comment]
Duplicate of #6079
